tests/unit_tests/test_cui_arguments.py

```diff
@@ -101,7 +101,7 @@ def test_nonexistent_user_file(self, fs: FakeFilesystem, capsys):
         assert exc.value.code == 2
 
         captured = capsys.readouterr()
-        assert "error: fab file does not exist" in captured.err
+        assert "fab file does not exist" in captured.err
```

Comment:
Where we are testing error messages I think we should test as much as possible. i.e. prefer equality over "in".